modules/core/src/v2/internal/tssUtils.ts

```diff
@@ -181,7 +236,7 @@ export class TssUtils {
     const bitgoKey = await openpgp.readKey({ armoredKey: bitgoPublicKeyStr });
 
     const userToBitGoMessage = await openpgp.createMessage({
-      text: Buffer.from(userKeyShare.yShares[3].u).toString('hex'),
+      text: Buffer.concat([Buffer.from(userKeyShare.yShares[3].u, 'hex'), Buffer.alloc(32)]).toString('hex'),
```

Comment:
Why here buffer.alloc(32)

Reply:
HSM expects this value to have the trailing empty 32 bits (bytes?).
without it, the keyshares do not get combined properly during wallet creation, and then signing won't work.

i merged a change recently to fix this independently.